mmv1/products/firebase/terraform.yaml

```diff
@@ -57,6 +57,28 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           - project
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/self_link_as_name.erb
+  AppleApp: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["projects/{{project}}/iosApps/{{appId}}", "iosApps/{{appId}}", "{{appId}}"]
+    autogen_async: true
+    skip_sweeper: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "firebase_apple_app_basic"
+        min_version: "beta"
+        primary_resource_id: "basic"
+        vars:
+          display_name: "Display Name Basic"
+        test_env_vars:
+          org_id: :ORG_ID
+          project_id: :PROJECT_NAME
+        test_vars_overrides:
+          bundle_id: '"apple.app." + randString(t, 5)'
+          app_store_id: 'randInt(t)'
+          team_id: "9987654321" # Has to be a 10-digit number.
```

Comment:
does something like 'randInt(t, 10)' work here?  not sure which part of the ecosystem is resolving the random numbers...

Reply:
@mraouffouad please respond to this question and remove the randomness if it's not necessary.

Reply:
If the randomness is not necessary, these lines could even be removed and you could just rely on the primary setting of these variables in `vars` (once that's added.)